src/wp-includes/script-loader.php

```diff
@@ -1254,7 +1265,10 @@ function wp_default_scripts( $scripts ) {
 
 		$scripts->add( 'farbtastic', '/wp-admin/js/farbtastic.js', array( 'jquery' ), '1.2' );
 
-		$scripts->add( 'iris', '/wp-admin/js/iris.min.js', array( 'jquery-ui-draggable', 'jquery-ui-slider', 'jquery-touch-punch' ), '1.1.1', 1 );
+		$scripts->add( 'iris', '/wp-admin/js/iris.min.js', array( 'jquery-ui-widget' ), '1.1.1', 1 );
```

Comment:
Is changing the color picker not covered in another PR:
https://github.com/ClassicPress/ClassicPress/pull/1300

Reply:
Yes and no. In that PR, the dependency is changed to `jquery-ui-core`, which is correct after the other changes made in that PR. In this PR, though, I have abstracted out the one section of that file's code and put it in the code for the `jquery-ui-widget`, which used to be where it was in the first place. (Core currently still lists the widget file but redirects to the `jquery-ui-core` file.)

The alternative would be to leave the `jquery-ui-widget` file with just the widget code in it and continue to have the `jquery-ui-widget` file redirect to it, but I thought that that seemed unnecessarily messy.